build.gradle

```diff
@@ -36,7 +37,6 @@ configurations.all {
 ext {
     // Play Services is a big download so it should only be upgraded if necessary and after
     // community discussion and a warning period.
-    playServicesVersion = '10.0.1'
```

Comment:
why did you remove this? It's good to have variables to store versions that are used multiple times or  you removed it since there's alot of version matching that has to be taking place to get all the dependencies to work well?

Reply:
This is an anti-pattern starting with play services 15.0.0: https://developers.google.com/android/guides/versioning#remove_old_anti-pattern

Reply:
Okay! Didn't know that.